### ISSUE TITLE
ctable error: attempt to set 'rownames' on an object with no dimensions

### ISSUE THREAD

**Issue by chris-dworschak** [dcomtois/summarytools#145]
Awesome package, thank you!

I hope I'm not overlooking anything obvious here, but whenever I include the `prop = "n"` argument in `ctable()`, I'm encountering the error 

```
Error in `rownames<-`(x, value) : 
  attempt to set 'rownames' on an object with no dimensions
```

and sometimes the same for "colnames". Below a MWE to reproduce the rownames error:

```
set.seed(121)
ctable(rpois(100,2), rpois(100,2))
ctable(rpois(100,2), rpois(100,2), prop = "n")
```
Cheers!

**Comment by dcomtois** [comment]
Hi, I confirm... will issue a fix as soon as possible. Thanks for reporting this.

**Comment by dcomtois** [comment]
Note that in the meantime, you can do something like this:

```r
a <- as.factor(rpois(100,2))
b <- as.factor(rpois(100,2))
ctable(a,b,prop="n")

```

**Comment by dcomtois** [comment]
Fixed in v1.0.0
